api/nakadi-event-bus-api.yaml

```diff
@@ -904,6 +1158,26 @@ definitions:
         description: |
           Offset of the event being pointed to.
 
+  CursorCommitResult:
+    description: |
+      The result of single cursor commit. Holds a cursor itself and a result value.
+    required:
+      - cursor
+      - result
+    properties:
+      cursor:
+        $ref: '#/definitions/Cursor'
+      result:
+        type: string
+        enum:
+          - committed
+          - outdated
```

Comment:
This enum can't be extended without a major revision of the api (unexpected enum values break clients). Could the values be described as strings instead?